docs/guides/how_to_guides/creating_batches/how_to_configure_a_runtime_data_connector.rst

```diff
@@ -0,0 +1,80 @@
+.. _how_to_guides__creating_batches__how_to_configure_a_runtime_data_connector:
+
+How to configure a RuntimeDataConnector
+==========================================
+
+This guide demonstrates how to configure a :ref:`RuntimeDataConnector<runtime_data_connector_and_runtime_batch_request>`.
+
+.. admonition:: Prerequisites: This how-to guide assumes you have already:
+
+    - :ref:`Set up a working deployment of Great Expectations <tutorials__getting_started>`
+    - :ref:`Understand the basics of Datasources in 0.13 or later <reference__core_concepts__datasources>`
```

Comment:
I've noticed the use of 0.13 in a bunch of other docs, that should be changed to V3 (Batch Request) API.